Sources/SourceKitLSP/Swift/SwiftLanguageServer.swift

```diff
@@ -253,7 +253,7 @@ public actor SwiftLanguageServer: ToolchainLanguageServer {
 
     var newDiags: [CachedDiagnostic] = []
     sourcekitdDiagnostics?.forEach { _, diag in
-      if let diag = CachedDiagnostic(diag, in: snapshot, useEducationalNoteAsCode: supportsCodeDescription) {
+      if let diag = CachedDiagnostic(diag, in: snapshot, useEducationalNotificationAsCode: supportsCodeDescription) {
```

Comment:
This is about notes, not notifications, so `useEducationalNoteAsCode` is correct.